### DISCUSSION TITLE
Master is again updated... and 2019 Scouting repository created

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#159] [Team Posts]
I've merged the pull requests from Matt (adding the DriveClimber class for the rear-lift) and Patrick (cleanup). If you've reviewing code, pick up these changes, please.

I've also created (at Matt's request) the Scouting 2019 Repository. For now everyone can read, but only the scouting code team members can write to that repository. I know Coach Chappell and Glushyn are eager to have our scouting software fully ready for the season as soon as we can have it, so this should be discussed tomorrow with Danny (I will not be able to attend tomorrow).

Thanks everyone for your patience and participation today. I heard kind of mixed reviews on the day overall, but it sounds like a lot of people found benefit in the review, so I hope it was worth your time.

-Chris

**Comment by MikasaSucasa** [comment]
By tomorrow, Chris meant Monday/Today. Unfortunately, I have late meetings and you all are being visited by Paddock so no mentors will be attending. Upperclassmen, you should be able to "escape" and continue working. I do not know how many of you I can say that to but you are all welcome to work if the teachers allow you. If you all work and any of the upperclassmen think that you are being a distraction or unfocused, I am comfortable with them using their discretion and asking anyone to leave. This is not to be harsh but we need to stay focused as we have very little done.
I am available by phone. Upperclassmen, feel free to contact Chris for my work number (248-xxx-xxxx) or email me directly.
Tasks to be completed:
1) Bosch seat motor wiring needs to be complete on the practice bot ASAP.
2) DriveStraight2Angle needs to be functioning for auton (use gyro)
3) Rotate2Angle needs to be functioning for auton (doesn't need to be PID, just close enough)
3) Play with counters for bosch seat motor after wiring. This needs to be working for our hatch mech.
4) lowest priority* Drive Encoders need to be re-tightened (one is yellow and one is red status leds). This can be done by the build team
5) Electrical Team, there are a ton of harnessing that needs to be done. before connectors are fully seated, an inspection by me or Chris will be required... This means no soldering and covering anything going on the comp bot without an inspection.
6) Button logic should be figured out for gamepad... i.e. when in manual mode, the buttons just do positions, when in auto, we do auto-assist... when interrupted, we interrupt the command. Lots of buttons and logic that is complicated. This will take a while but needs to be done prior to unbag

**Comment by JooshO** [comment]
Status update as of the end of Monday:
DriveStraight to angle and rotate to angle are completed and functioning. DriveToAngle still requires a more fleshed out exit case.
Motor Controller wiring for bosch was not finished when we left (Not functional, I'm not sure how far done it got)
The drivetrain is continuing to deteriorate

**Comment by coverbeek** [comment]
Let’s discuss the test bot drivetrain tomorrow (Tuesday). I will be able to be at practice from about 4:30-6:45. We don’t want to base too much of our tuning on a broken drivetrain. If we need build help to replace/repair the belt drive, let’s get that arranged.
